### ISSUE TITLE
Create index.html files

### ISSUE THREAD

**Issue by stefanoborini** [wolever/pip2pi#104]
It is my understanding that dir2pi just deploys packages in a simple pypi structure, but relies on autoindex to provide the html file that pip uses. Is there an option to generate (and keep up to date, if new packages are added) the index file? my current server creates a fancy javascript interface to represent directory content, but will show an index.html if present.

**Comment by tom-shan** [comment]
dir2pi will create an index.html in each directory by default refer to https://github.com/wolever/pip2pi/blob/master/libpip2pi/commands.py#L220-L226

But I think https://github.com/wolever/pip2pi/blob/master/libpip2pi/commands.py#L376 this line should be changed to `pkg_index += "<a href='%s/index.html'>%s</a><br />\n" %(`

**Comment by stefanoborini** [comment]
@tom-shan yes, the problem is that it doesn't only do that. I would need a functionality just to create the index file from the currently deployed packages, but I could not find anything that does so.

**Comment by tom-shan** [comment]
IIUC, if you really only want those index.htmls, how about deleting all the useless packages after the generation?

**Comment by stefanoborini** [comment]
@tom-shan that's not what I want. I have a pypi content with all the packages there and they have to stay there, but I don't have autoindex in the web server to generate the A HREF page. I need something to generate this page.
